src/main.cpp

```diff
@@ -14,6 +14,8 @@
 #include "core/hle/libraries/libs.h"
 #include "core/linker.h"
 #include "emuTimer.h"
+#include <core/file_sys/fs.h>
```

Comment:
```suggestion
#include "core/file_sys/fs.h"
```
Project headers use quotes